platform/ios/MGLMapView.mm

```diff
@@ -1032,10 +1032,11 @@ - (BOOL)gestureRecognizer:(UIGestureRecognizer *)gestureRecognizer shouldRecogni
 
 - (void)trackGestureEvent:(NSString *)gesture forRecognizer:(UIGestureRecognizer *)recognizer
 {
+    CGPoint ptInView = CGPointMake([recognizer locationInView:recognizer.view].x, [recognizer locationInView:recognizer.view].y);
+    const CLLocationCoordinate2D coord = [self convertPoint:ptInView toCoordinateFromView:recognizer.view];
```

Comment:
I'm not sure that it can be put into a non-main UI thread, as the `convertPoint` method relies on `[UIView convertPoint:fromView]` itself.